provider/provider.go

```diff
@@ -26,6 +26,12 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("HOST", nil),
 				Description: "Only set this value if you are using a specific URL given by leanspace",
 			},
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("REGION", nil),
```

Comment:
Could we pass the default region here and not in client.go ?

Reply:
We could, I'll investigate this